core/src/test/java/org/apache/calcite/jdbc/CalciteRemoteDriverTest.java

```diff
@@ -880,6 +880,40 @@ public Service create(AvaticaConnection connection) {
     assertThat(updateCount, is(1));
   }
 
+  @Test public void testInsertBatchWithPreparedStatement() throws Exception {
+    final Connection connection = DriverManager.getConnection(
+        "jdbc:avatica:remote:factory="
+            + LocalServiceModifiableFactory.class.getName());
+    assertThat(connection.getMetaData().supportsBatchUpdates(), is(true));
+    assertThat(connection.isClosed(), is(false));
```

Comment:
What is the point of these assertions?

Reply:
To check that the `connection` is correct.

Reply:
How is it related to CALCITE-3338?

Reply:
I don't quite understand what you mean. 
This case comes from [CALCITE-3338](https://issues.apache.org/jira/browse/CALCITE-3338)

Reply:
`connection.isClosed()` behavior is not altered in CALCITE-3338, so I do not see why do you test it here.

Please avoid test duplication. If you want to test `connection.isClosed()`, then create a dedicated test for it.

Reply:
ok, update the code.